applications/coreutils/mkdir.cpp

```diff
@@ -28,9 +28,8 @@ static CommandLine cmdline = CMDLINE(
 void print_path(const char *path)
 {
     if (verbose)
-    {
```

Comment:
Can you put those curly braces back please ?

Reply:
I have removed them to test that clang-tidy works as it should. When I make it work I will undo this change.

Reply:
Ok no problem, You should write in on the description of the pr :sweat_smile: